src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const promise1 = new Promise((resolve) => {
+  const logo = document.querySelector('.logo');
+
+  logo.addEventListener('click', () => resolve());
```

Comment:
Consider adding error handling for the case where the logo element is not found. This will prevent runtime errors if the element is missing.